src/crosschain/chainUtils/tron.ts

```diff
@@ -77,11 +78,11 @@ export function prepareTronTransaction({
 
 const ADDRESS_PREFIX_REGEX = /^(41)/
 
-export function tronAddressToEvm(address: string): string {
+export function tronAddressToEvm(address: string): EvmAddress {
```

Comment:
change type `string` to `Address`

Reply:
I've decided to keep `string` here because there are too many places where ethers' `BaseContract.address` is used which has `string` type.